core/arch/arm/mm/core_mmu.c

```diff
@@ -643,7 +649,8 @@ bool core_pbuf_is(uint32_t attr, paddr_t pbuf, size_t len)
 	case CORE_MEM_SEC:
 		return pbuf_is_inside(secure_only, pbuf, len);
 	case CORE_MEM_NON_SEC:
-		return pbuf_is_inside(nsec_shared, pbuf, len);
+		return pbuf_is_inside(nsec_shared, pbuf, len) ||
+			pbuf_is_inside(ddr, pbuf, len);
```

Comment:
In this case we will need support from `plat-` code. I think it can be look like
`register_nsec_ddr()` similar to `register_phys_mem()`.

And we will need this for every platform. 

OP-TEE will disable dynamic SHM and registered SHM feature in `tee_entry_exchange_capabilities()` if platform does not register nsec DDR. What do you think about this approach?